plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -6,15 +6,37 @@ import { Button, Tooltip } from '@wordpress/components';
 import { getNewPath } from '@woocommerce/navigation';
 import { help } from '@wordpress/icons';
 import { Table } from '@woocommerce/components';
+import { useContext, useEffect, useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { getAdminSetting } from '~/utils/admin-settings';
+import { getAdminSetting } from '../../../utils/admin-settings';
 import { Subscription } from './types';
 import './my-subscriptions.scss';
+import { MarketplaceContext } from '../../contexts/marketplace-context';
+import { fetchSubscriptions } from '../../../marketplace/utils/functions';
 
 export default function MySubscriptions(): JSX.Element {
+	const [ subscriptions, setSubscriptions ] = useState<
+		Array< Subscription >
+	>( [] );
+	const marketplaceContextValue = useContext( MarketplaceContext );
+	const { setIsLoading } = marketplaceContextValue;
```

Comment:
I inline these to save a line of code. Not a major thing but I can't help myself. 

```suggestion
	const { setIsLoading } = useContext( MarketplaceContext );
```